Solutions/Endpoint Threat Protection Essentials/Hunting Queries/BackupDeletion.yaml

```diff
@@ -1,6 +1,8 @@
 id: 56ebae61-89cf-42d9-99f4-3dff8ba33885
 name: Backup Deletion 
 description: |
+    'This query detects attempts to delete backups, which could be legitimate or ransomware actions to prevent file restoration and disrupt business services.'
```

Comment:
This query detects attempts to delete backups, which could be ransomware activity to prevent file restoration and disrupt business services.

Reply:
Resolved